cred/src/main/webapp/WEB-INF/web.xml

```diff
@@ -101,8 +101,12 @@
                 ca.nrc.cadc.vosi
             </param-value>
         </init-param>
+        <init-param>
```

Comment:
this no longer needs to be configured; LogControlServlet always reads `cadc-log.properties`

that is already documented in the README